### DISCUSSION TITLE
Hidden + Freeze Frame causes hit circles to approach at the same time

### DISCUSSION THREAD

**Discussion by kingmigdor** [ppy/osu#21127] [Q&A]
### Type

Game behaviour

### Bug description

It's likely impossible to see those circles approach since all circles disappear at the same time. Hit circles should rather disappear when you need to click them than disappear when the first circle needs to be clicked.

### Screenshots or videos

https://user-images.githubusercontent.com/67754395/199788511-54cd6b67-363e-43ad-aba5-dbdca2baf8b2.mp4

### Version

2022.1101.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9931468/database.log)
[input.log](https://github.com/ppy/osu/files/9931469/input.log)
[network.log](https://github.com/ppy/osu/files/9931473/network.log)
[performance.log](https://github.com/ppy/osu/files/9931474/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9931475/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9931478/updater.log)

**Comment by mk56-spn** [comment]
this was intentional on my part atleast, and the general reception has been good towards the hidden combination, with people saying it provides a fun challenge, i myself have enjoyed learning maps with HDFR

**Comment by Doxxz** [comment]
Doesn't Hidden's "Only fade approach circles" setting mitigate this already?

**Comment by kingmigdor** [comment]
> this was intentional on my part atleast, and the general reception has been good towards the hidden combination, with people saying it provides a fun challenge, i myself have enjoyed learning maps with HDFR

You cannot really see when to click the circle, that's the thing

**Comment by mk56-spn** [comment]
> > this was intentional on my part atleast, and the general reception has been good towards the hidden combination, with people saying it provides a fun challenge, i myself have enjoyed learning maps with HDFR
> 
> You cannot really see when to click the circle, that's the thing

I don't think this is an issue. They can be hit correctly through a mix of rythmic understanding of the map, memory and pattern recognition. Not everything has to be  sight-readable

**Comment by bdach** [comment]
Judging from the reactions on the opening post I don't think this is an unanimous bug that 100% needs addressing. Converting to discussion until consensus is reached one way or another.

**Comment by Walavouchey** [comment]
will add that FR without HD almost had a similar reading difficulty aspect to FRHD since it changed approach rate to make the effect of circles popping in more obvious, but now that non-HD has been made a lot easier to read, i can understand why the jump in difficulty between FR and HDFR can seem jarring.

either way, the way it works is very intentional as the mod is supposed to be challenging to read

**Reply by mk56-spn** [reply]
id also like to add that theres still the inbetween of HDFR( fade approach circles only ) for those that wish for something inbetween in difficulty, or FRTC, theres no shortage of intermediate options
